pkg/relayer/relayminer.go

```diff
@@ -121,13 +121,13 @@ func (rel *relayMiner) ServePprof(ctx context.Context, addr string) error {
 	// If no error, start the server in a new goroutine
 	go func() {
 		rel.logger.Info().Str("endpoint", addr).Msg("starting a pprof endpoint")
-		server.ListenAndServe()
+		_ = server.ListenAndServe()
```

Comment:
Should we do something here if there's an error?

Reply:
Let's log the error.